### PR TITLE
Aws funk

### PR DESCRIPTION
**Pull request by Piwington**

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ functions:
       exclude:
         - ./**
     layers:
-      - arn:aws:lambda:eu-west-2:${self:custom.accountId}:layer:dev-es-commonfunctions:14
+      - arn:aws:lambda:eu-west-2:${self:custom.accountId}:layer:dev-es-common-functions:1
```
Comment (thomashenson) [inline serverless.yml:36]: Why this been changed back to 1?

Comment (thomashenson) [reply serverless.yml:36]: Ah. So from further inspection appears to be because the name has been changed from 'funk'. Can someone confirm this, please?

**Review by KrisRogos — APPROVED** [review]

File: serverless.yml (hunk 1 above)
Comment (KrisRogos) [reply serverless.yml:36]: When David renamed the function it reset the counter back to 1

Comment (Piwington) [reply serverless.yml:36]: As I added a - to the name for standards the new one which gets deployed will start at 1 again.

**Review by thomashenson — APPROVED** [review]

**Pull request merged**